Starting backup...
Backup complete: C:\Users\QB-Admin\majool-inventory-system\inventory_backend\backups\2025-06-17
Backup complete: C:\Users\QB-Admin\majool-inventory-system\inventory_backend\backups\2025-06-17
Scheduler started: Sync every 1 min, Backup at 4 PM
Scheduler started: Sync every 1 min, Backup at 4 PM
INFO:     192.168.1.35:57217 - "GET /scanner/ping HTTP/1.1" 200 OK
INFO:     192.168.1.35:57222 - "POST /scanner/login HTTP/1.1" 200 OK
INFO:     192.168.1.35:57224 - "GET /scanner/products HTTP/1.1" 200 OK
INFO:     192.168.1.35:57225 - "GET /scanner/noser-units HTTP/1.1" 200 OK
INFO:     192.168.1.35:57226 - "GET /scanner/brands HTTP/1.1" 200 OK
INFO:     192.168.1.35:57227 - "GET /scanner/noser-units HTTP/1.1" 200 OK
INFO:     192.168.1.35:57228 - "GET /scanner/brands HTTP/1.1" 200 OK
INFO:     192.168.1.35:57229 - "GET /scanner/master-skus HTTP/1.1" 200 OK
INFO:     192.168.1.35:57230 - "GET /scanner/master-skus HTTP/1.1" 200 OK
INFO:     192.168.1.35:57231 - "GET /scanner/brands HTTP/1.1" 200 OK
INFO:     192.168.1.35:57232 - "GET /scanner/categories HTTP/1.1" 200 OK
INFO:     192.168.1.35:57237 - "POST /scanner/add-delivery HTTP/1.1" 200 OK
INFO:     192.168.1.35:57239 - "GET /scanner/noser-units HTTP/1.1" 200 OK
INFO:     192.168.1.35:57240 - "GET /scanner/brands HTTP/1.1" 200 OK
User 1 assigning serial 'TESTSN2' -> unit 585
INFO:     192.168.1.35:57249 - "POST /scanner/assign-serial HTTP/1.1" 200 OK
INFO:     192.168.1.35:57250 - "GET /scanner/noser-units HTTP/1.1" 200 OK
INFO:     192.168.1.35:57251 - "GET /scanner/brands HTTP/1.1" 200 OK
User 1 assigning serial 'TESTSN2' -> unit 584
INFO:     192.168.1.35:57425 - "POST /scanner/assign-serial HTTP/1.1" 400 Bad Request
ERROR in /handle-return-scan: (psycopg2.errors.InsufficientPrivilege) permission denied for sequence returns_return_id_seq

[SQL: 
                INSERT INTO returns (
                    original_unit_id, product_id, serial_number, serial_assigned_at,
                    assigned_by_user_id, po_number, sn_prefix, sold
                ) VALUES (
                    %(original_unit_id)s, %(product_id)s, %(serial_number)s, %(serial_assigned_at)s,
                    %(assigned_by_user_id)s, %(po_number)s, %(sn_prefix)s, %(sold)s
                )
            ]
[parameters: {'original_unit_id': 585, 'product_id': 30, 'serial_number': 'TESTSN2', 'serial_assigned_at': datetime.datetime(2025, 6, 17, 14, 51, 49, 299379), 'assigned_by_user_id': 1, 'po_number': 'TEST', 'sn_prefix': None, 'sold': True}]
(Background on this error at: https://sqlalche.me/e/20/f405)
INFO:     192.168.1.35:57429 - "POST /scanner/handle-return-scan HTTP/1.1" 500 Internal Server Error
